models/time_driven_static_connection.h

```diff
@@ -0,0 +1,161 @@
+/*
+ *  time_driven_static_connection.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+#ifndef TIME_DRIVEN_STATIC_CONNECTION_H
+#define TIME_DRIVEN_STATIC_CONNECTION_H
+
+/* @BeginDocumentation
+   Name: time_driven_static_synapse - Synapse type for time-driven static
+   connections.
+
+   Description:
+
+   For efficiency reasons spiking connections in NEST are only updated on every
+   presynaptic spike, making it difficult to implement time-driven plasticity
+   rules. This model implements a simple approach to time-driven synapse
+   updates: the framework for continuous interactions is used to communicate
+   spikes. This causes this model to be updated in every time step.
+
+   The event received by the synapse contains a buffer of length h / dt in which
+   non-zero values indicate a spike. For simplicity this static synapse model
+   forwards the entire TimeDrivenSpikeEvent to the receiving node where the
+   buffer is unpacked. To compute updates in dt steps in the synapse object, the
+   event should instead be unpacked in `send()` and a SpikeEvent in every dt
+   step should be forwarded to the receiving node.
+
+   time_driven_static_synapse does not support any kind of plasticity. It simply
+   stores
```

Comment:
Just a minor formatting issue (probably due to auto-formatting): Maybe move the remaining part of the sentence up to this line. Having a line with a single word within a sentence looks a bit strange.

Reply:
fixed